src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private final int firstSide;
+    private final int secondSide;
+
+    public Rectangle(String color, int firstSide, int secondSide) {
+        super(color);
+        this.firstSide = firstSide;
+        this.secondSide = secondSide;
+    }
+
+    @Override
+    public double getArea() {
+        return firstSide * secondSide;
+    }
+
+    @Override
+    public void toDrawFigure() {
```

Comment:
The method name `toDrawFigure` does not follow the Java naming conventions and does not align with the checklist item that suggests not to put all behavior into a single interface if the methods are conceptually different. Consider renaming it to `draw` to align with the checklist, and ensure it's in a separate interface if required by the project structure.